pyecsca/sca/attack/CPA.py

```diff
@@ -45,7 +46,7 @@ def compute_intermediate_value(self, guessed_scalar: int, target_bit: int, point
         result = ctx.actions.get_by_index([0, action_index])[0]
         return result.output_points[0].X
 
-    def compute_correlation_trace(self, guessed_scalar: int, target_bit: int) -> list[any]:
+    def compute_correlation_trace(self, guessed_scalar: int, target_bit: int) -> list[Any]:
```

Comment:
The correlation trace is definitely not a list of `Any`, proper number type (float?) should be used.